yarn-project/noir-contracts/src/contracts/noir-aztec3/src/abi.nr

```diff
@@ -19,6 +19,8 @@ struct PrivateContextInputs {
     roots: CommitmentTreesRoots,
 
     contract_deployment_data: ContractDeploymentData,
+
+    private_global_variables: PrivateGlobalVariables,
```

Comment:
I think these should belong in the `call_context`, rather than being in their own section?

Reply:
Was not put in the `call_context` as those values could change throughout execution, e.g., `msg_sender` etc.

Seemed better to have the constants separately to that.